modules/administration_manual/pages/configuration/user/user_auth_ldap.adoc

```diff
@@ -667,13 +661,11 @@ server
 * The certificate is announced in the system’s LDAP configuration file,
 usually `/etc/ldap/ldap.conf`.
 
-* Using LDAPS, also make sure that the port is correctly configured (by
-default 636)
+* Using LDAPS, also make sure that the port is correctly configured (by default 636)
 * If you get the error "_Lost connection to LDAP server_" or
 "**No connection to LDAP server**" double check the connection parameters and
 try connecting to LDAP with tools like `ldapsearch`. If using ldaps or
```

Comment:
Please capitalise LDAPS, as you have done with LDAP.

Reply:
just to let you know, I did not change the content but linebreaking in the source for better readablity...